api/controllers/UserController.js

```diff
@@ -1506,8 +1466,25 @@ module.exports = {
       user = await AuthPolicy.isTOTPValid(user, token);
     }
 
+    // If an emailId parameter was present, we will use that value. However, if
+    // nothing was sent, we need to determine the ID of the user's primary email
+    // address of the as a default value.
```

Comment:
> address of the as a default value

Some words seem to be missing.

Reply:
I left that in there to see if you're paying attention 😉 

thanks for the spot!